docs/docs/references/terminology.md

```diff
@@ -0,0 +1,182 @@
+# Terminology
+
+This page explains the terminology system used in Trivy, helping users understand the specific terms and concepts unique to the Trivy ecosystem.
+
+## Core Concepts
+
+### Target
+Types of artifacts that Trivy can scan, like container images and filesystem.
+
+### Scanner
+Trivy's built-in security scanning engines. Trivy has four main scanners:
+
+- [Vulnerability Scanner](../scanner/vulnerability.md): Detects known vulnerabilities in OS packages and application dependencies
+- [Misconfiguration Scanner](../scanner/misconfiguration/index.md): Identifies security misconfigurations in IaC files
+- [Secret Scanner](../scanner/secret.md): Finds hardcoded secrets and sensitive information
+- [License Scanner](../scanner/license.md): Identifies software license issues
+
+### Scan Assets
+External data that Trivy downloads and uses during scanning:
+
+- [Vulnerability Database (Trivy DB, trivy-db)](#vulnerability-database-trivy-db-trivy-db)   : Database containing vulnerability information
+- [Java Index Database (Trivy Java DB, trivy-java-db)](#java-index-database-trivy-java-db-trivy-java-db): Database for Java artifact identification
+- [Checks Bundle (trivy-checks)](#checks-bundle): Archive containing misconfiguration detection rules
+- [VEX Repository](#vex-repository)
+
+## Vulnerability Scanning
+
+### Vulnerability Database (Trivy DB, trivy-db)
+The core vulnerability database required for vulnerability detection.
+Contains comprehensive vulnerability information for multiple ecosystems.
+Distributed via OCI registry.
+
+Managed at [the GitHub repository](https://github.com/aquasecurity/trivy-db).
+
+### Java Index Database (Trivy Java DB, trivy-java-db)
+Specialized database used for identifying Java libraries and their components during JAR/WAR/PAR/EAR scanning.
+Distributed via OCI registry.
+
+Managed at [the GitHub repository](https://github.com/aquasecurity/trivy-java-db).
+
+### vuln-list
+A GitHub repository that collects and stores vulnerability information from various data sources.
+This repository serves as the foundation for building the Trivy DB.
+
+Managed at:
+
+- https://github.com/aquasecurity/vuln-list
+- https://github.com/aquasecurity/vuln-list-nvd
+- https://github.com/aquasecurity/vuln-list-redhat
+- https://github.com/aquasecurity/vuln-list-debian
+- etc.
```

Comment:
Can you make them as hyperlinks?

Reply:
done https://github.com/aquasecurity/trivy/pull/7996/commits/306d6b021420c49b1774de525651ce0f58245808